docs/dev/starting-a-mod.md

```diff
@@ -24,37 +24,63 @@ It does come with some minor drawbacks such as it using "LethalCompanyTemplate"
+First things first, you'll need to create your project. If you've not done so already, we recommend running the following command in a console to add some Lethal Company templates for new projects:
 
-```cmd
-dotnet new -i BepInEx.Templates --nuget-source https://nuget.bepinex.dev/v3/index.json
+```shell
+dotnet new -i Xilophor.LCModTemplates
 ```
 
 Next, you'll want to create a new project (sometimes called "solution", in CSharp). There are two main ways to do this.
 
-#### Using an IDE (more control) {#using-ide}
+#### Using an IDE (easier) {#using-ide}
 
-Depending on your IDE, this process will look slightly different. You'll want to give the solution the name of your soon-to-be mod. If given the option to use a template (you may want to google for *"how to use template in Visual Studio"* or *"how to use template in Rider"*), use the `BepInEx 5 Plugin Template`.
+Depending on your IDE, this process will look slightly different. You'll want to give the solution the name of your soon-to-be mod. If given the option to use a template (you may want to google for *"how to use template in Visual Studio"* or *"how to use template in Rider"*), use the `Lethal Company Harmony Mod Template`.
 
-#### Using the console (simpler) {#using-console}
+::: warning
+Note that Visual Studio has issues with both the `NuGetPackages` setting and the `UseNetcodePatcher` setting. Anything set in `NuGetPackages` will be ignored by Visual Studio; and if you set `UseNetcodePatcher` to true, you will have to manually run the `install-netcode-patcher.cmd` file.
+
+Rider also does not support custom arguments/settings in templates at the moment, so you will not be able to customize your mod other than the mod's name.
+:::
+
+#### Using the console (recommended for control) {#using-console}
 
-Alternatively, you can open a console and run the following command, assuming you've set up the templates using the command above. Replace `MyFirstPlugin` with your mod's name:
+Alternatively, you can open a console and run the following command, assuming you've set up the templates using the command above.
+Replace `MyFirstMod` with your mod's name, and `MyName` with your username:
 
-```cmd
-dotnet new bepinex5plugin -n MyFirstPlugin -T "netstandard2.1" -U "2022.3.9"
+```shell
+dotnet new lchmod -n MyFirstMod -M MyName.MyFirstMod
 ```
 
-::: warning
-Some people have been [reporting problems](https://github.com/BepInEx/BepInEx.Templates/issues/8) creating new BepInEx plugins from the template when using the .NET 8 SDK:
+To see the other options provided in the mod template, you can use the following command:
+
+```shell
+dotnet new lchmod --help
+```
+
+#### Note for MonoMod users {#monomod-note}
 
-`Failed to create template.
-Details: Object reference not set to an instance of an object.`
+If you want to use MonoMod, you can instead use the `Lethal Company MonoMod Mod Template` in your IDE or use `lcmmod` instead of `lchmod` in the console.
```

Comment:
Okay yeah, it might be a good idea. I can't think of any method provided by VitePress to have two separate guides, other than maybe two separate details containers.

Reply:
What does a detail container look like?

Reply:
Hmm... that might be too much to toss into a container.

Not entirely sure what the best solution is here.